frontend/src/component/admin/users/UsersHeader/UsersHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, styled } from '@mui/material';
+import { InviteLinkBar } from '../InviteLinkBar/InviteLinkBar';
+import { useUiFlag } from 'hooks/useUiFlag';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { LicensedUsersBox } from './LicensedUsersBox';
+
+interface StyledContainerProps {
+    licensedUsersEnabled: boolean;
+}
+
+const StyledContainer = styled(Box)<StyledContainerProps>(
+    ({ theme, licensedUsersEnabled }) => ({
+        display: 'grid',
+        gridTemplateColumns: licensedUsersEnabled ? '60% 40%' : '100%',
```

Comment:
We can also use fractions here, which is more idiomatic for grid (_gridiomatic_?) (but this comment is most important because of that awful pun, so do what you will)

```suggestion
        gridTemplateColumns: licensedUsersEnabled ? '3fr 2fr' : '100%',
```